### DISCUSSION TITLE
Is it possible to emulate an MP100 Module in future?

### DISCUSSION THREAD

**Discussion by alexnet1977** [emsesp/EMS-ESP32#2155] [Ideas]
I already use the RC100H Emulation with my Loxone System to Send Room Temperature and Humidy to the RC310 of my WLW196i. May it possible in Future to Emulate a MP100 Pool Module the same way? We are planning to have a pool, that should be heated with the WLW196 and Photovoltaic in Summer. The original Module just controlls a 3 Way Valve, give the binary demand of the Pool Controller and the actual Temperature of the Pool to the RC310 and monitors a max Temperature Switch. Nothing more...

**Comment by MichaelDvP** [comment]
Actually we know only the broadcasted monitor telegram 0x5BA from MP100 with only3 values. 
For emulation we need a full communication protocol and config values.
